### PR TITLE
ceph: fixes for "tell <service>.*" command

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
@Liuchang0812

File: src/ceph.in [hunk 1]
```diff
@@ -1020,13 +1023,11 @@ def main():
     # of the form 'cmdNNN' followed by an array of argument descriptors)
     # as part of the validated argument JSON object
 
-    targets = [target]
-
     if target[1] == '*':
```
Comment (Liuchang0812) [inline src/ceph.in]: manual tested. target is ` (u'mds', u'*') or (u'osd', 0)`. it's ok,

File: src/ceph.in [hunk 2]
```diff
@@ -199,6 +201,14 @@ def mgrids():
     return l
 
 
+def ids_by_service(service):
+    ids = {"mon": monids,
+           "osd": osdids,
+           "mds": mdsids,
+           "mgr": mgrids}
+    return ids[service]();
```
Comment (tchaikov) [inline src/ceph.in]: remove the trailing semicolon.

**Comment by tchaikov** [comment]
changelog:

- removed the trailing `;`

**Comment by Liuchang0812** [comment]
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/crushtool/build.t: failed
--- /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/crushtool/build.t
+++ /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/crushtool/build.t.err
@@ -14,12 +14,6 @@
 # display a warning if there is more than one root
 #
   $ crushtool --outfn "$map" --build --num_osds 5 node straw 2 rack straw 1 
-  .* The crush rulesets will use the root rack0 (re)
-  and ignore the others.
-  There are 3 roots, they can be
-  grouped into a single root by appending something like:
-    root straw 0
-  
 #
 # crush rulesets are generated using the OSDMap helpers
 #
```

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-09-06_14:17:17-rados-wip-sage-testing-2017-09-05-2251-distro-basic-smithi/

**Pull request merged**